samples/boards/rpi_pico/uart_pio/boards/rpi_pico.overlay

```diff
@@ -0,0 +1,30 @@
+&pinctrl {
+	pio1_uart_default: pio1_uart_default {
+		pio1_uart_rx: group1 {
+			#pin-cells = <0>;
+			pinmux = <PIO1_P1>;
+			input-enable;
+			bias-pull-up;
+		};
+		pio1_uart_tx: group2 {
+			#pin-cells = <0>;
+			pinmux = <PIO1_P0>;
+		};
+	};
+};
+
+&pio1 {
+ 	status = "okay";
+
+	pinctrl-0 = <&pio1_uart_default>;
+	pinctrl-names = "default";
+	state-machine-num = <2>;
+
+	pins = <&pio1_uart_rx>, <&pio1_uart_tx>;
```

Comment:
The right hand side value represent pinmux, not pin.
IMO, The property name should change to `pinmuxs`.

Reply:
done